docs-src/typescript/how-to-replay-a-workflow-execution-in-typescript.md

```diff
@@ -9,7 +9,45 @@ tags:
   - typescript
 ---
 
-Retrieve Event History with [`WorkflowService.getWorkflowExecutionHistory`](https://typescript.temporal.io/api/classes/proto.temporal.api.workflowservice.v1.workflowservice-1/#getworkflowexecutionhistory).
+Use [worker.runReplayHistories](https://typescript.temporal.io/api/classes/worker.Worker#runreplayhistories)
+or [worker.runReplayHistory](https://typescript.temporal.io/api/classes/worker.Worker#runreplayhistory)
+to replay multiple or one Workflow Histories.
```

Comment:
```suggestion
To replay one or more Event Histories, use [worker.runReplayHistories](https://typescript.temporal.io/api/classes/worker.Worker#runreplayhistories) or [worker.runReplayHistory](https://typescript.temporal.io/api/classes/worker.Worker#runreplayhistory).
```
Or should the sentence begin "To replay a Workflow Execution, use..."?

Reply:
I dunno, that's a great question. In this context they are, in some sense, equivalent. I think replaying the history reads a bit more obviously, though.